### ISSUE TITLE
You missed something when you changed to bigger data types

### ISSUE THREAD

**Issue by disastranagant** [hw1-5/C-Hero-Calc#7]
WB damage calc is now capped at 32768, a slight improvement over flat 32000.

**Comment by hw1-5** [comment]
No, this is intentional. When the calc tracks mid-battle info, it saves opponent's hp as short. Changing this data type will inflate memory usage greatly for every single calculation, so what I did was change a few things here and there so that compiling a version which calculates bigger damage is easier.

If you use ageum and want that extra bit of edge over your peers, change the data type of DamageType from cosmosData.h, which by default is short, to int64_t or long long and recompile it.
